controllers/host/platformnetworks.go

```diff
@@ -337,6 +370,22 @@ func (r *HostReconciler) ReconcileAddrPoolResource(client *gophercloud.ServiceCl
 
 	validation_result := r.ValidateAddressPool(network_instance, addrpool_instance, system_info)
 
+	// Secondary stacks for cluster-* should always be configured
```

Comment:
Can you describe the issue to avoid here?

Reply:
Done.